kettleweb/templates/rollout_view.html

```diff
@@ -3,21 +3,46 @@
 {% block head %}
 {{ super() }}
 {% if rollout.rollout_start_dt and not (rollout.rollout_finish_dt or rollout.rollback_start_dt) %}
-<script type="text/JavaScript">
-    window.auto_refresh = true;
-    function auto_refresh_fn(){
-        if (window.auto_refresh == true){
-            location.reload(true);
-        }
-    };
-    setTimeout("auto_refresh_fn()", 1000);
+<script type="text/javascript">
+    var refreshInterval;
 
+    function clearRefresh(){
+      clearInterval(refreshInterval);
+      return false;
+    }
+
+    function refreshContentInner() {
+        $.get(
+            window.location + "",
```

Comment:
I'm regretting being such a hack, we should probably use `window.location.toString()`